Makefile

```diff
@@ -8,6 +8,7 @@ mypy:
 	mypy --ignore-missing-imports $(MODULES)
 
 test: lint mypy
+	tests/fixtures/populate.py --s3-bucket $(DSS_S3_BUCKET_TEST_FIXTURES) --gs-bucket $(DSS_GS_BUCKET_TEST_FIXTURES)
```

Comment:
Well, #245 is approved, so the latter approach makes the most sense.

Nevertheless, I believe there is some peculiarity to the way you're using git branches...